lesson07/assignment/index.html/App.vue

```diff
@@ -0,0 +1,218 @@
+
+ <p>Nisi ad leo morbi faucibus accumsan nostra feugiat nullam elit eu id. Ultricies porta nam sem himenaeos. Aliquet fermentum ante ornare fringilla magna litora nisl montes nisi? Hendrerit vehicula magnis tellus fusce ante feugiat. Fermentum posuere imperdiet blandit risus, vel mi elit. Pulvinar lacus auctor interdum, mus facilisis imperdiet rhoncus viverra proin. Phasellus lectus penatibus elit venenatis vulputate consequat per integer. Eleifend facilisis hac augue facilisi cubilia integer porttitor aliquet. Nunc nisl per, tincidunt ultrices. Sollicitudin id senectus vitae semper ante dictum purus ullamcorper. Hac.</p>
+
+ <p>Ligula orci dignissim lobortis sit fringilla rutrum vivamus. Hac eros enim porttitor cubilia nulla suspendisse. Dui nunc facilisis commodo egestas quam vitae inceptos etiam purus hendrerit. Massa augue platea purus cursus. Convallis consectetur gravida a nibh quisque. Nullam, lorem netus cursus. Porttitor, amet non ad fermentum sociosqu metus! Vestibulum massa sociis ut parturient quisque porttitor vulputate odio curabitur integer imperdiet. Quisque lorem habitant donec quam est elementum lacus ante. Nec venenatis nibh habitasse litora mus.</p>
+
+ <p>Quam aliquet quis enim sit pellentesque vestibulum ridiculus. Fusce sed fringilla lectus mauris, consequat habitant? Quis fusce duis hac laoreet praesent natoque augue ultricies erat condimentum penatibus! Morbi hac tempus fringilla. Primis a congue taciti varius inceptos penatibus velit fermentum. Donec egestas gravida nunc augue himenaeos enim potenti tortor. Est pellentesque hendrerit tincidunt netus, sodales velit non pellentesque per maecenas vitae! Laoreet odio sem at venenatis litora class taciti potenti! Penatibus scelerisque penatibus nam cras ridiculus condimentum nostra class himenaeos vestibulum.</p>
+
+ <p>Posuere ipsum malesuada volutpat parturient. Iaculis duis tempor molestie viverra commodo hendrerit tincidunt mattis. Hac etiam eget ut condimentum. Gravida felis consequat, torquent at! Sagittis gravida quam purus odio maecenas scelerisque blandit gravida ante nisl. Quam hac ligula, felis vestibulum cras vitae. Tristique est purus arcu viverra varius venenatis sed netus. Nunc gravida sem!</p>
+
+<footer class="footer">
+  <ul>
+    <li><a href="./index.html">About</a><a href="./portfolioimages.html">Portfolio</a><a href="processaccordian.html">Process</a><a href="contactgrid.html">Book a Consultation</a></li>
+  </ul>
+</nav>
+<p><small>© 2022 Plant Portraits LLC | Seattle, WA</small></p>
+</footer>
+</div>
+</template>
+
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css"></script>
+
+<script>
+import app.vue from './vue.js/src/index.html/app.vue'
+
+export default {
+  name: 'App',
+  data() {
+  return {
+  links: ['Home', 'Portfolio', 'Process', 'Book a Consultation'],
+  show: true,
+  }
```

Comment:
Where is the `awesome` variable that's being used on line 6? I'd expect it to be declared here and given an initial value.